cves/CVE-2011-3192.yml

```diff
@@ -52,14 +56,10 @@ fixes:
      note:
    - commit: 58e50d77a98357b5cef0eb7b8694a165631ab87a
      note:
-   - commit: adf40b995fbb5d78a9f3880b682f3632cbaa1d7f
-     note:
    - commit: 1f36e470005b1fb8276c893a6458f7045d3af5c2
```

Comment:
Mark as a regression to the fix in commit `adf40b995fbb5d78a9f3880b682f3632cbaa1d7f`, not necessarily a fix itself

Reply:
Good suggestion. Moved and updated